integration_test.go

```diff
@@ -50,25 +46,29 @@ func TestMain(m *testing.M) {
 		fmt.Println("minio is up")
 		break
 	}
-	code := m.Run()
-	err = cmd.Process.Signal(os.Interrupt)
-	if err != nil {
-		panic(err)
-	}
-	io.Copy(os.Stderr, stderrPipe)
-	processState, err := cmd.Process.Wait()
-	if err != nil {
-		panic(err)
-	}
-	if processState.ExitCode() != 0 {
-		panic(fmt.Errorf("minio exited with code %d", processState.ExitCode()))
-	}
-	os.Exit(code)
 }
 
-const testLogSaysPastTheEnd = "oh no! we fell off the end of the log!"
+// cleanupContainer stops a running named container and removes its assigned
+// name. This is helpful in the event that a container wasn't properly killed
+// during a previous test run or if manual testing was being performed and not
+// cleaned up.
+func cleanupContainer() {
+	// Unconditionally stop the container.
+	_, _ = exec.Command("podman", "stop", containerName).Output()
+
+	// Unconditionally remove the container name if the operator did manual
+	// container testing, but didn't clean up the name.
+	_, _ = exec.Command("podman", "rm", containerName).Output()
+}
+
+func init() {
+	cleanupContainer()
+	startContainer()
```

Comment:
It's nice to avoid `init` functions when we can. Can you move this into `TestIntegration`?